docs/reference/workflows/monobehaviour/interaction/commands/world-commands.md

```diff
@@ -22,45 +22,30 @@ There are two types of commands in SpatialOS:
 
 The commands documentation is:
 
-* [MonoBehaviour world commands]({{urlRoot}}/reference/workflows/monobehaviour/interaction/commands/world-commands)
+* MonoBehaviour world commands - this document
 * [ECS world commands]({{urlRoot}}/reference/workflows/ecs/interaction/commands/world-commands)
 * [MonoBehaviour component commands]({{urlRoot}}/reference/workflows/monobehaviour/interaction/commands/component-commands)
-* ECS component commands - this document
+* [ECS component commands]({{urlRoot}}/reference/workflows/ecs/interaction/commands/component-commands)
 * Both workflows - [world commands API reference]({{urlRoot}}/api/core/commands/world-commands)
```

Comment:
Can we actually remove this little section? I feel like its not very useful and points you all over the shop.